OgreMain/include/OgrePlatform.h

```diff
@@ -73,6 +74,8 @@ THE SOFTWARE.
 #    define OGRE_CPU OGRE_CPU_ARM
 #elif defined( __mips__ ) || defined( __mips64 ) || defined( __mips64_ ) || defined( _M_MIPS )
 #    define OGRE_CPU OGRE_CPU_MIPS
+#elif defined( __loongarch64 )
```

Comment:
Bitness is not reflected here, so just `__loongarch__` will do.

```suggestion
#elif defined( __loongarch__ )
```